style-transfer/styleTransfer.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+function sizeOfShape(shape) {
+  return shape.reduce((a, b) => {
+    return a * b;
+  });
+}
+
+async function buildConstantByNpy(builder, url) {
+  const dataTypeMap = new Map([
+    ['f2', {type: 'float16', array: Uint16Array}],
+    ['f4', {type: 'float32', array: Float32Array}],
+    ['f8', {type: 'float64', array: Float64Array}],
+    ['i1', {type: 'int8', array: Int8Array}],
+    ['i2', {type: 'int16', array: Int16Array}],
+    ['i4', {type: 'int32', array: Int32Array}],
+    ['i8', {type: 'int64', array: BigInt64Array}],
+    ['u1', {type: 'uint8', array: Uint8Array}],
+    ['u2', {type: 'uint16', array: Uint16Array}],
+    ['u4', {type: 'uint32', array: Uint32Array}],
+    ['u8', {type: 'uint64', array: BigUint64Array}],
+  ]);
+  const response = await fetch(url);
+  const buffer = await response.arrayBuffer();
+  const npArray = new numpy.Array(new Uint8Array(buffer));
+  if (!dataTypeMap.has(npArray.dataType)) {
+    throw new Error(`Data type ${npArray.dataType} is not supported.`);
+  }
+  const dimensions = npArray.shape;
+  const type = dataTypeMap.get(npArray.dataType).type;
+  const TypedArrayConstructor = dataTypeMap.get(npArray.dataType).array;
+  const typedArray = new TypedArrayConstructor(sizeOfShape(dimensions));
+  const dataView = new DataView(npArray.data.buffer);
+  const littleEndian = npArray.byteOrder === '<';
+  for (let i = 0; i < sizeOfShape(dimensions); ++i) {
+    typedArray[i] = dataView[`get` + type[0].toUpperCase() + type.substr(1)](
+        i * TypedArrayConstructor.BYTES_PER_ELEMENT, littleEndian);
+  }
+  return builder.constant({type, dimensions}, typedArray);
+}
+
+/* eslint max-len: ["error", {"code": 130}] */
+
+// Style Transfer Baseline Model
+export class StyleTransfer {
+  constructor() {
+    this.model_ = null;
+    this.compiledModel_ = null;
+    this.dimensions_ = [1, 3, 540, 540];
+    this.constPow_ = null;
+    this.constAdd_ = null;
+  }
+
+  buildSubNetwork_(builder, conv2D, variableMul, variableAdd) {
```

Comment:
I think this is to build the instanceNormalization. Please name the function as that. And could you please also try the builder.instanceNormalization operation. It would be good to keep the two code paths to show the different usages.

Reply:
It seems this comment not to be addressed in the new commit. Did I miss anything?

Reply:
Sorry, I missed this comment. :p

Could you please guide me how to map these variables to builder.instanceNormalization?